### PR TITLE
Method createApiToken

### PR DESCRIPTION
**Pull request by wzalazar**
This PR introduces

- new version 0.1.3
- adds method createApiToken

### CODE REVIEW

File: src/Frost.ts [hunk 1]
```diff
@@ -312,6 +312,28 @@ export class Frost {
...
+
+    const request = fetch(`${this.host}${Path.TOKENS}`, options)
+
+    return Promise.race([request, this.timeoutPromise()])
+      .then(async (value: any) => {
+        if (value.ok) return await value.json()
+
+        throw await value.text()
+      })
+      .catch(e => {
```
Comment (lautarodragan) [inline src/Frost.ts]: What's the purpose of `catch`ing and then `throw`ing?

Comment (wzalazar) [reply src/Frost.ts]: What do you think?

Comment (lautarodragan) [reply src/Frost.ts]: Yeah but if you're doing nothing with the error you caught, then why catch it?

Comment (wzalazar) [reply src/Frost.ts]: You're right, I don't need the catch. I will remove it. 👍

**Pull request merged**